scripts/sommelier

```diff
@@ -217,6 +217,18 @@ start_app() {
     fi
     "${WINELOADER}" explorer /desktop="virtual","${native_res}" "${app}" "${ACTION_ARGS[@]}"
     local exit_code="$?"
+  elif [[ -f "${ACTION_ARGS}" ]]; then
+    # Passing args to wine apps
+    for i in "${ACTION_ARGS[@]}"; do
```

Comment:
I think a better approach is to put this for loop around line 202 and just edit/overwrite the ACTION_ARGS array. This way, the logic also works for apps which are launched on a virtual desktop (line 218).

Reply:
ah I didn't thought of that some how missed it.